achievements/theGodfatherConsigliere.achievement.js

```diff
@@ -0,0 +1,21 @@
+var _ = require('lodash');
+var console = process.console;
```

Comment:
please remove the `var console = process.console;` line. you don't console log here so it's not in use